### PR TITLE
ADEN-5298 Prepare callbacks for after page render event

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [mercury.run.js#L17](https://github.com/Wikia/app/blob/7236021580eb224f500a061fd80fd3980fbe08b8/extensions/wikia/AdEngine/js/run/mercury.run.js#L17): Function has 14 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![INFO][INFO] [mercury.run.js#L48](https://github.com/Wikia/app/blob/7236021580eb224f500a061fd80fd3980fbe08b8/extensions/wikia/AdEngine/js/run/mercury.run.js#L48): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [mercury.run.js#L17](https://github.com/Wikia/app/blob/cfdbb50de413303805eb4d74294bf742fe0014c2/extensions/wikia/AdEngine/js/run/mercury.run.js#L17): Function has 14 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![INFO][INFO] [mercury.run.js#L48](https://github.com/Wikia/app/blob/cfdbb50de413303805eb4d74294bf742fe0014c2/extensions/wikia/AdEngine/js/run/mercury.run.js#L48): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by nandy-andy — APPROVED** [review]

File: extensions/wikia/AdEngine/js/run/mercury.run.js [hunk 1]
```diff
@@ -62,16 +75,15 @@ require([
...
-		}
-
-		if (
-			geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneCountries) &&
-			geo.isProperGeo(instantGlobals.wgAdDriverRubiconFastlaneMercuryFixCountries)
-		) {
-			rubiconFastlane.call();
-		}
-	});
+	// TODO: Remove else statement, this step is required in order to keep bidders working during cache invalidation
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/run/mercury.run.js]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 1 major
* ![INFO][INFO] 2 info

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [mercury.run.js#L17](https://github.com/Wikia/app/blob/828c2cbcf9df1f1337604684761b49502a77b6e1/extensions/wikia/AdEngine/js/run/mercury.run.js#L17): Function has 14 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![INFO][INFO] [mercury.run.js#L61](https://github.com/Wikia/app/blob/828c2cbcf9df1f1337604684761b49502a77b6e1/extensions/wikia/AdEngine/js/run/mercury.run.js#L61): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**